### ISSUE TITLE
process.env.toJSON is incorrectly returned

### ISSUE THREAD

**Issue by OmgImAlexis** [oven-sh/bun#1540]
### What version of Bun is running?

0.2.2

### What platform is your computer?

Darwin 20.6.0 Darwin Kernel Version 20.6.0: Tue Jun 21 20:50:28 PDT 2022; root:xnu-7195.141.32~1/RELEASE_X86_64 x86_64 i386

### What steps can reproduce the bug?

```
➜  bun-test git:(main) ✗ ls -a
.             ..            .git          .gitignore    README.md     bun.lockb     node_modules  package.json  src           tsconfig.json
➜  bun-test git:(main) ✗ cat src/index.ts 
console.log(process.env);
➜  bun-test git:(main) ✗ env -i ~/.bun/bin/bun run ./src/index.ts
{ toJSON: toJSON { } }
```

### How often does it reproduce? Is there a required condition?

100% of the time

### What is the expected behavior?

`{ }`

### What do you see instead?

`{ toJSON: toJSON { } }`

### Additional information

Original issue: https://discord.com/channels/876711213126520882/1044495237432741959/1044495237432741959

p.s. sorry for the bad title

**Comment by Electroid** [comment]
@robobun 

```js
console.log(process.env);
```

**Comment by robobun** [comment]
@Electroid here you go!
```tsx
{ HOME: "/tmp/bun-84PDu6", PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin", toJSON: toJSON { } }
```
<details>
  <summary>Code</summary>
  
```tsx
console.log(process.env);
```
</details>

_Ran using the latest build of Bun, an all-in-one JavaScript runtime._

**Comment by Electroid** [comment]
Fixed in [51d0c3b](https://github.com/oven-sh/bun/commit/51d0c3b79f63e5ca1965cb5280c58fc0a9bc4f73)
